logic/src/commonMain/kotlin/com/wire/kalium/logic/data/e2ei/E2EIRepository.kt

```diff
@@ -244,4 +246,16 @@ class E2EIRepositoryImpl(
     override suspend fun nukeE2EIClient() {
         e2EIClientProvider.nuke()
     }
+
+    override suspend fun getCurrentClientDomainCRL(): Either<CoreFailure, ByteArray> =
+        userConfigRepository.getE2EISettings().flatMap {
+            wrapApiRequest {
+                acmeApi.getClientDomainCRL(it.discoverUrl)
```

Comment:
hint: the discovery url will have the directory inside it, so we need to have it like this: (Url(it.discoverUrl).protocolWithAuthority)